manager/apis/app/v1alpha1/m4dmodule_types.go

```diff
@@ -38,14 +38,15 @@ const (
 	Read ModuleFlow = "read"
 )
 
-// CredentialManagementType indicates whether this module queries the SecretProvider by itself to get
-// credentials, or whether it assumes that the data-mesh will inject them.
+// CredentialManagementType indicates how this module gets the credentials:
+// In HashicorpVault option the module should use Hashicorp Vault client to retrieve the credentials.
+// Automatic option assumes that the data-mesh will inject the credentials to the modules.
```

Comment:
I thought we decided to get rid of this setting.

Reply:
Was not sure about that... I can remove it ...

Reply:
Yes. We will introduce an alternative when we get to chaining modules in the future.